scripts/hid_configurator/README.rst

```diff
@@ -1,55 +1,78 @@
-:orphan:
+.. _config_channel_script:
 
-.. _configuration_channel:
-
-Configuration channel for nRF52 Desktop
-#######################################
+HID configurator
+################
 
 Overview
 ********
 
-This application is an example of controlling nRF52 Desktop firmware parameters at runtime.
+This Python application is an example of using :ref:`config_channel` for communication between computer and firmware.
+It controls :ref:`nrf_desktop` firmware parameters at runtime.
 
-It uses HID feature reports to communicate with firmware.
+The script can be used to:
 
-Right now, it can be used to perform DFU or change following parameters:
+* perform DFU
+* configure following parameters at runtime:
 
-* mouse optical sensor resolution (CPI)
-* optical sensor power saving modes:
-	* downshift time from "Run" to "Rest1" mode
-	* downshift time from "Rest1" to "Rest2" mode
-	* downshift time from "Rest2" to "Rest3" mode
+	* mouse optical sensor resolution (CPI)
+	* mouse optical sensor power saving modes:
 
-Supported transports:
+		* downshift time from "Run" to "Rest1" mode
+		* downshift time from "Rest1" to "Rest2" mode
+		* downshift time from "Rest2" to "Rest3" mode
 
-* USB
-* BLE (Windows 10, Linux)
+To learn the details of the architecture of the firmware module, see :ref:`config_channel`.
 
 Usage
 *****
-On Windows:
 
-	py -3 configurator.py config sensor cpi 12000
-	py -3 configurator.py config sensor downshift_rest1 fetch
+To set value of sensor CPI:
+
+.. parsed-literal::
+   :class: highlight
+
+   python3 configurator.py config sensor cpi 12000
+
+To fetch some value, use `fetch` instead of numeric value.
+
+.. parsed-literal::
+   :class: highlight
+
+   python3 configurator.py config sensor downshift_rest1 fetch
+
+To perform DFU operation:
+
+.. parsed-literal::
+   :class: highlight
 
-On Linux:
+   python3 configurator.py dfu signed-dfu-image.bin
 
-	python3 configurator.py config sensor cpi 12000
-	python3 configurator.py config sensor downshift_rest1 fetch
+To get a list of possible arguments at each subcommand level, call with `--help` option.
```

Comment:
after build it shows _-help_, you may want to e.g. escape each character in double dash